src/Symfony/Component/Security/Core/Authentication/Token/AbstractToken.php

```diff
@@ -317,6 +317,19 @@ private function hasUserChanged(UserInterface $user)
             return true;
         }
 
+        $userRoles = \array_map('strval', (array) $user->getRoles());
```

Comment:
You need to remove all backslashes before function calls.

Reply:
removed